connector_importer/components/odoorecord.py

```diff
@@ -191,6 +191,8 @@ def odoo_write(self, values, orig_values):
         odoo_record = self.odoo_find(values, orig_values).with_context(
             **self.write_context()
         )
+        if "product_tmpl_id" not in orig_values and "product_tmpl_id" in odoo_record._fields.keys() and odoo_record.product_tmpl_id:
```

Comment:
this commit mentions `c_importer_product` but is done on the core module. It cannot stay here, this is specific to products. It must be fixed in the product mapper.

Reply:
I created an override of the `odoo_pre_write` function in "product.product.handler" component. The docstring of the function is stating  """Do some extra stuff before updating an existing object.""", it seams to be a great place to add the logic. Let me know what do you think